spec/sunomono_test.rb

```diff
@@ -0,0 +1,431 @@
+            to include('android', 'break_build_if_failed.sh', 'check_if_tests_failed.sh', 'ios')
+        expect(Dir.entries("#{@project_name}/config/scripts/android")).
+            to include('run_tests_all_devices.sh', 'start_emulators.sh', 'stop_emulators.sh')
+        expect(Dir.entries("#{@project_name}/config/scripts/ios")).
+            to include('.', '..', 'build_app.rb', 'build_app.yml', 'devices.txt', 'run_tests_all_devices.sh')
+      end
+
+      it 'Create new project using alias command' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        expect(Dir.entries(@project_name)).
+            to include('screenshots', 'features', 'config', 'README.md', 'Gemfile', '.gitignore')
+        # Features files
+        expect(Dir.entries("#{@project_name}/features")).
+            to include('android', 'ios', 'step_definitions', 'support')
+        expect(Dir.entries("#{@project_name}/features/android")).
+            to include('android_screen_base.rb', 'features', 'screens', 'step_definitions', 'support')
+        expect(Dir.entries("#{@project_name}/features/ios")).
+            to include('ios_screen_base.rb', 'features', 'screens', 'step_definitions', 'support')
+        expect(Dir.entries("#{@project_name}/features/step_definitions")).
+            to include('base_steps.rb')
+        expect(Dir.entries("#{@project_name}/features/support")).
+            to include('env.rb', 'exceptions.rb')
+
+        # Config files
+        expect(Dir.entries("#{@project_name}/config")).
+            to include('cucumber.yml', 'email', 'load_classes.rb', 'scripts')
+
+        # Email files
+        expect(Dir.entries("#{@project_name}/config/email")).
+            to include('template.html')
+
+        # Scripts files
+        expect(Dir.entries("#{@project_name}/config/scripts")).
+            to include('android', 'break_build_if_failed.sh', 'check_if_tests_failed.sh', 'ios')
+        expect(Dir.entries("#{@project_name}/config/scripts/android")).
+            to include('run_tests_all_devices.sh', 'start_emulators.sh', 'stop_emulators.sh')
+        expect(Dir.entries("#{@project_name}/config/scripts/ios")).
+            to include('.', '..', 'build_app.rb', 'build_app.yml', 'devices.txt', 'run_tests_all_devices.sh')
+      end
+    end
+
+    context 'Try creates new project with a invalid argument' do
+      it 'Project will not be generated' do
+        system "suno new '#{@project_name}' invalid argument", :out => File::NULL
+
+        expect(Dir.entries(".")).
+            not_to include("#{@project_name}")
+      end
+    end
+
+    context 'Should return the latest version of sunonomo' do
+      it 'Returns the latest version' do
+        system "sunomono version", :out => File::NULL
+        version = "#{Sunomono::VERSION}"
+
+        expect(version).
+            to end_with('0.2.0.pre')
+      end
+
+
+      it 'Should return the latest version of sunonomo using alias command' do
```

Comment:
The alias `suno` is made with 

```
require 'sunomono'
Sunomono::SunomonoRunner.start
```

So if `sunomono` works, `suno` works too. I think that only one test with the alias is OK to ensure that it works